### DISCUSSION TITLE
please add all the mod settings

### DISCUSSION THREAD

**Discussion by BigDogg99** [Docile-Alligator/Infinity-For-Reddit#1189] [Duplicate Ideas] [answered]
this is the only reason i still have reddit app and i cant get rid of it

**Comment by Wladefant** [comment, marked as answer]
Duplicate of #948

**Reply by BigDogg99** [reply]
are you planing to add it??

**Reply by Wladefant** [reply]
That is definitely a big and needed feature but it is unclear when and how

**Reply by BigDogg99** [reply]
this is only reason i cant switch from official app
